API/RecommendationService/__init__.py

```diff
@@ -19,14 +19,24 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
         command_list = get_param_str(req, 'command_list')
         if not command_list:
             return func.HttpResponse('Illegal parameter: please pass in the parameter "command_list"', status_code=400)
-    except ValueError:
+    except ValueError as e:
         return func.HttpResponse('Illegal parameter: the parameter "command_list" must be the type of string', status_code=400)
 
     try:
         top_num = get_param_int(req, 'top_num')
     except ValueError:
         return func.HttpResponse('Illegal parameter: the parameter "top_num" must be the type of int', status_code=400)
```

Comment:
Or we can verify that when the recommended type is `mix`, `top_num` cannot be sent, but `command_top_num` and `scenario_top_num` should be sent

Reply:
> Or we can verify that when the recommendation type is `mix`, `top_num` cannot be sent, but `command_top_num` and `scenario_top_num` should be sent

I think it makes sense. Or perhaps we could remove the `top_num` parameter and set default value of `command_top_num` and `scenario_top_num` to 5, for example, so that the previous version of `az next` won't raise an exception.